lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -258,3 +264,61 @@ workflow:
     - if: $CI_COMMIT_TAG # if this is a release
     - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH # if this is a push to develop
     - if: $CI_PIPELINE_SOURCE == "web" # if triggered from the Gitlab UI
+
```

Comment:
Nit, but please move the `workflow` to the very bottom of this file so it is easy to find since that is where it is typically placed.